src/com/iesam/AlonsoMadrigal/domain/models/Clase_ordenadores.java

```diff
@@ -0,0 +1,22 @@
+package com.iesam.AlonsoMadrigal.domain.models;
+
+public class Clase_ordenadores extends Clase{
+    private Integer nºordenadores;
+    private String tipo_ordenadores;
+
+    public Integer getNºordenadores() {
```

Comment:
de verdad que habéis puesto el circulito de número ??? Por favor, nada de símbolos.